src/lib.rs

```diff
@@ -116,3 +119,5 @@ mod assert_obj_safe;
 mod assert_trait;
 mod assert_type;
 mod const_assert;
+#[doc(hidden)]
+pub mod type_level_bool;
```

Comment:
Please add a comment stating this this module (and its contents) should *never* be used publicly and is not part of this crate's semver requirements.